### ISSUE TITLE
Multiple closed cloops

### ISSUE THREAD

**Issue by Ktysai** [wojiaojiao/pegasus_gazebo_plugins#2]
Hello Angel,

Nice workaround!

It is possible to add multiple closed loops?

All the best!

**Comment by wojiaojiao** [comment]
yes! this pluging just like button that connect any two links you want to connect
